main/integration-tests/__test__/api-tests/studies/create-study.test.js

```diff
@@ -121,5 +149,35 @@ describe('Create study scenarios', () => {
         code: errorCode.http.code.badRequest,
       });
     });
+
+    it('should fail if internal guest creates an unregistered BYOB study', async () => {
```

Comment:
Why do we say here `unregistered` BYOB study? Also add a test case for researcher?